### PR TITLE
Add more detail to page rendering and add links to specs.

### PR DESCRIPTION
**Pull request by samlh**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -55,7 +55,7 @@ converts the HID usage into a scancode. In this case the scan code is
 ``VK_RETURN`` (``0x0D``). The ``KBDHID.sys`` driver interfaces with the
 ``KBDCLASS.sys`` (keyboard class driver). This driver is responsible for
 handling all keyboard and keypad input in a secure manner. It then calls into
-``Win32K.sys`` (after potentially passing the message through 3rd party
```
Comment (dtracers) [inline README.rst]: I do believe this is a revert.

Comment (samlh) [inline README.rst]: Fixed in https://github.com/samlh/what-happens-when/commit/707ec21e1e3742bc74686782be979aeb9026127e

**Comment by samlh** [comment]
The travis failure is for a term from the css spec, so cannot be addressed.

**Comment by alex** [comment]
That's pretty obnoxious! Can you just remove the `--weasel`` flag for now.

**Pull request merged**